edx_api/enrollments/models_test.py

```diff
@@ -0,0 +1,146 @@
+"""Models Tests for the Enrollment API"""
+
+import os.path
+import json
+from unittest import TestCase
+
+from .models import (
+    Enrollments,
+    Enrollment,
+    CourseDetails,
+    CourseMode,
+)
+
+
+class EnrollmentsTests(TestCase):
+    """Tests for enrollments object"""
+    @classmethod
+    def setUpClass(cls):
+        with open(os.path.join(os.path.dirname(__file__),
+                               'fixtures/user_enrollments.json')) as file_obj:
+            cls.enrollments_json = json.loads(file_obj.read())
+
+        cls.enrollments = Enrollments(cls.enrollments_json)
+
+    def test_enrollments(self):
+        """Test the enrolled_courses method"""
+        enrollments_list = list(self.enrollments.enrolled_courses)
+        assert len(enrollments_list) == len(self.enrollments_json)
+
+        for elem in enrollments_list:
+            assert isinstance(elem, Enrollment)
+
+
+class EnrollmentTests(TestCase):
+    """Tests for enrollment object"""
+    @classmethod
+    def setUpClass(cls):
+        with open(os.path.join(os.path.dirname(__file__),
+                               'fixtures/user_enrollments.json')) as file_obj:
+            cls.enrollments_json = json.loads(file_obj.read())
+
+        cls.enrollment_json = cls.enrollments_json[0]
+        cls.enrollment = Enrollment(cls.enrollment_json)
+
+    def test_created(self):
+        """Test for created property"""
+        assert self.enrollment.created == self.enrollment_json['created']
```

Comment:
I would make this assert (and the ones below) the actual values of the json payload, so we can ensure the implementation is actually correct.